### DISCUSSION TITLE
Weblate change null to "None" after commit

### DISCUSSION THREAD

**Discussion by WhoisUnknown** [WeblateOrg/weblate#10481] [Q&A] [answered]
I'm trying to support null value in my JSON files. All my untranslated keys are set to "None" after adding new translations and push changes to Github

Can I change this behavior?

<img width="382" alt="image" src="https://github.com/WeblateOrg/weblate/assets/36919008/579de464-302f-48d0-b146-40fc127b868e">

I'm using self-hosted latest version Weblate by Docker

**Comment by WhoisUnknown** [comment, marked as answer]
I updated from 4x to 5x weblate and problem is solved

**Reply by nijel** [reply]
https://github.com/translate/translate/pull/5034 was the fix for this.
